Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/META-INF/persistence.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\" ?>\r\n<persistence xmlns=\"http://xmlns.jcp.org/xml/ns/persistence\" version=\"2.2\">\r\n    <persistence-unit name=\"my-emf\">\r\n        <class>jpa.entities.Student</class>\r\n        <properties>\r\n            <property name=\"hibernate.dialect\" value=\"org.hibernate.dialect.MySQLDialect\"/>\r\n            <property name=\"hibernate.connection.url\" value=\"jdbc:mysql://localhost:3306/school_jpa?createDatabaseIfNotExist=true\"/>\r\n            <property name=\"hibernate.connection.username\" value=\"root\"/>\r\n            <property name=\"hibernate.connection.password\" value=\"12345\" />\r\n            <property name=\"hibernate.hbm2ddl.auto\" value=\"update\" />\r\n            <property name=\"hibernate.show_sql\" value=\"true\" />\r\n            <property name=\"hibernate.format_sql\" value=\"true\" />\r\n        </properties>\r\n    </persistence-unit>\r\n</persistence>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/META-INF/persistence.xml b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/META-INF/persistence.xml
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/META-INF/persistence.xml	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/META-INF/persistence.xml	(date 1751139442504)
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <persistence xmlns="http://xmlns.jcp.org/xml/ns/persistence" version="2.2">
     <persistence-unit name="my-emf">
-        <class>jpa.entities.Student</class>
+        <class>lab.jpa.entities.Student</class>
         <properties>
             <property name="hibernate.dialect" value="org.hibernate.dialect.MySQLDialect"/>
             <property name="hibernate.connection.url" value="jdbc:mysql://localhost:3306/school_jpa?createDatabaseIfNotExist=true"/>
Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/hibernate/student.hbm.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\" ?>\r\n<!DOCTYPE hibernate-mapping PUBLIC\r\n        \"-//Hibernate/Hibernate Mapping DTD 3.0//EN\"\r\n        \"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd\">\r\n\r\n<hibernate-mapping>\r\n    <class name=\"hibernate.entities.Student\" table=\"students\">\r\n        <id name=\"id\" column=\"student_id\">\r\n            <generator class=\"identity\"></generator>\r\n        </id>\r\n        <property name=\"name\" column=\"student_name\" />\r\n\r\n    </class>\r\n</hibernate-mapping>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/hibernate/student.hbm.xml b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/hibernate/student.hbm.xml
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/hibernate/student.hbm.xml	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/resources/hibernate/student.hbm.xml	(date 1751139442523)
@@ -4,7 +4,7 @@
         "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 
 <hibernate-mapping>
-    <class name="hibernate.entities.Student" table="students">
+    <class name="lab.hibernate.entities.Student" table="students">
         <id name="id" column="student_id">
             <generator class="identity"></generator>
         </id>
Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/entities/Student.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/entities/Student.java b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/entities/Student.java
rename from ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/entities/Student.java
rename to ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/entities/Student.java
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/entities/Student.java	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/entities/Student.java	(date 1751139442458)
@@ -1,4 +1,4 @@
-package hibernate.entities;
+package lab.hibernate.entities;
 
 public class Student {
     private long id;
Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/entities/Student.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/entities/Student.java b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/entities/Student.java
rename from ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/entities/Student.java
rename to ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/entities/Student.java
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/entities/Student.java	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/entities/Student.java	(date 1751139442437)
@@ -1,4 +1,4 @@
-package jpa.entities;
+package lab.jpa.entities;
 
 import jakarta.persistence.*;
 
Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/Main.java b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/Main.java
rename from ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/Main.java
rename to ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/Main.java
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/hibernate/Main.java	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/hibernate/Main.java	(date 1751139442490)
@@ -1,6 +1,6 @@
-package hibernate;
+package lab.hibernate;
 
-import hibernate.entities.Student;
+import lab.hibernate.entities.Student;
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.cfg.Configuration;
Index: ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/Main.java b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/Main.java
rename from ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/Main.java
rename to ../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/Main.java
--- a/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/jpa/Main.java	(revision 0bd26730f8f6b1d816124a1abe85ee2c950489a6)
+++ b/../../Spring_Boot_Course/Introduction_to_Hibernate/src/main/java/lab/jpa/Main.java	(date 1751139442472)
@@ -1,6 +1,6 @@
-package jpa;
+package lab.jpa;
 
-import hibernate.entities.Student;
+import lab.hibernate.entities.Student;
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.EntityManagerFactory;
 import jakarta.persistence.Persistence;
